server/src/server.js

```diff
@@ -287,7 +289,7 @@ const getProcess = (recordInfo) => {
     case 'GStreamer':
       return new GStreamer(recordInfo);
     case 'FFmpeg':
-    default:    
```

Comment:
unneeded space.

Reply:
Origional code has extra space at the end of the line, my editor auto trim them